web/admin/src/containerHeader.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 DoubleDutch, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { memo } from 'react'
+import { translate as t } from '@doubledutch/admin-client'
+import Select from 'react-select'
+import ModIcon from './modicon'
+import PresentationDriver from './PresentationDriver'
+
+const ContainerHeader = props => {
```

Comment:
It's customary to name the file the same as the React component (same capital casing)